src/pages/settings/SettingsPage.tsx

```diff
@@ -85,6 +87,14 @@ export const SettingsPage = () => {
         }
     };
 
+    const stickyFooter = (
```

Comment:
would rename that as something like "savePanel" or so as its not the actual footer